remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/filter/JPAContinuousQueryCacheEventFilterConverterFactory.java

```diff
@@ -20,41 +13,16 @@
  * @since 8.0
  */
 @NamedFactory(name = JPAContinuousQueryCacheEventFilterConverterFactory.FACTORY_NAME)
-@MetaInfServices
-public final class JPAContinuousQueryCacheEventFilterConverterFactory implements CacheEventFilterConverterFactory {
+@MetaInfServices(value = CacheEventFilterConverterFactory.class)
```

Comment:
This is what I meant: ``@MetaInfServices(CacheEventFilterConverterFactory.class)``